pom.xml

```diff
@@ -888,7 +888,7 @@
         <carbon.kernel.registry.imp.pkg.version.range>[1.0.1, 2.0.0)</carbon.kernel.registry.imp.pkg.version.range>
 
         <!-- Carbon Identity Framework version -->
-        <carbon.identity.framework.version>5.20.16</carbon.identity.framework.version>
+        <carbon.identity.framework.version>5.20.38-SNAPSHOT</carbon.identity.framework.version>
```

Comment:
Need to update the version once the PR: https://github.com/wso2/carbon-identity-framework/pull/3505 is merged.